code/game/turfs/simulated/floor/plating/asteroid.dm

```diff
@@ -68,7 +68,23 @@
 			if(W.use_tool(src, user, 40, volume=50))
 				if(!can_dig(user))
 					return TRUE
-				to_chat(user, "<span class='notice'>You dig a hole.</span>")
+
+				if(W.tool_behaviour == TOOL_MINING && prob(2)) //I forgot about better mining tools. Brought down the probability and making the options more likely to be boring
+					to_chat(user, "<span class='warning'>You opened a fissure! A gas is escaping...</span>")
+					icon_state = "fissure"
+					icon_plating = "fissure"
+					environment_type="fissure"
+					switch(rand(1,10)) //This probably needs to be adjusted - picks between different types of fissures
```

Comment:
also pickweight would be better than this

Reply:
Oh hey pickweight is a thing. Testing it now, rand was admittedly pretty hacky

Reply:
Corrected in the last commit